settings

```diff
@@ -70,6 +67,8 @@ GPG_EXPIRE="1y"
 
 load_settings
 
+PASS_NAME_GPG="theforeman/releases/foreman/$FOREMAN_VERSION-gpg"
+PASS_NAME_KEY="theforeman/releases/foreman/$FOREMAN_VERSION-key"
```

Comment:
This overrides the settings, which means you can't override it anymore as we do:
```console
releases/pulpcore/settings
2:PASS_NAME_GPG="theforeman/releases/pulpcore/$VERSION-gpg"
3:PASS_NAME_KEY="theforeman/releases/pulpcore/$VERSION-key"

releases/foreman-debian/settings
1:PASS_NAME_GPG="theforeman/releases/foreman/debian-$VERSION-gpg"
2:PASS_NAME_KEY="theforeman/releases/foreman/debian-$VERSION-key"

releases/candlepin/settings
1:PASS_NAME_GPG="theforeman/releases/candlepin/$VERSION-gpg"
2:PASS_NAME_KEY="theforeman/releases/candlepin/$VERSION-key"
```

It's also not like we can use `$PROJECT` like this:
```suggestion
PASS_NAME_GPG="theforeman/releases/$PROJECT/$VERSION-gpg"
PASS_NAME_KEY="theforeman/releases/$PROJECT/$VERSION-key"
```

What I'm leaning to is move these definitions into `releases/foreman/settings` (and not have any default) but I don't think we load that now when inheriting settings.

Alternatively you can use:
```suggestion
PASS_NAME_GPG="${PASS_NAME_GPG:-theforeman/releases/foreman/$FOREMAN_VERSION-gpg}"
PASS_NAME_KEY="${PASS_NAME_KEY:-theforeman/releases/foreman/$FOREMAN_VERSION-key}"
```

Reply:
dang.

so the problem I was trying to fix is:
When signing Katello, `$OSES` ends up `''` as it first loaded `katello/4.14/settings`, then `foreman/3.12/settings` and then overwrote `OSES` with the value in `settings`.
To fix that, I moved the "load inherited foreman stuff" logic into `load_settings`, but that that meant that `FOREMAN_VERSION` is not yet populated when the old location of `PASS_NAME_*` was used, so I moved that too.

Reply:
I added a commit that I think addresses the concerns. It's not pretty, but I think it'll work.